app/assets/scripts/components/indicators/indicator-co2.js

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Prose from '../../styles/type/prose';
+import Gridder from '../../styles/gridder';
+import InpageHGroup from '../../styles/inpage-hgroup';
+import { Fold, FoldDetails } from '../../styles/fold';
+import {
+  IntroLead
+} from '../../styles/datasets';
+import media from '../../styles/utils/media-queries';
+
+import { glsp } from '../../styles/utils/theme-values';
+import Heading from '../../styles/type/heading';
+
+const LeadFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${media.largeUp`
+    padding-bottom: ${glsp(3)};
+  `}
+`;
+
+const IntroFold = styled(Fold)`
+  padding-bottom: 0;
+
+  ${Gridder} {
+    align-items: center;
+  }
+
+
+
+  ${FoldDetails} {
+    grid-column: content-start / content-end;
+
+    ${media.mediumUp`
+      grid-column: content-start / content-8;
+    `}
+
+    ${media.largeUp`
+      grid-column: content-start / content-10;
+    `}
+
+  }
+`;
+
+const ResearchFold = styled(Fold)`
+  ${Gridder} {
+    align-items: center;
+  }
+
+  /* stylelint-disable-next-line */
+  ${InpageHGroup} {
+    grid-row: 1;
+    grid-column: content-start / content-7;
+  }
+
+  ${Prose} {
+    grid-column: content-start / content-end;
```

Comment:
Textual content should not span the full page length. `content-10` for large up is enough.
This fold can actually use the same structure as Credits for example. Or the 2 column setup like NO2.

Reply:
this is updated.